### ISSUE TITLE
No default config

### ISSUE THREAD

**Issue by willdurand** [Zoapp/backend#9]
The code expects `src/config.js` to exist and to be a valid configuration file. Currently, we ignore this file and `yarn lint` outputs an error because the file may not exist. We should provide a default config file or handle optional config file differently.

**Comment by mikbry** [comment]
Is this bug fixed ?

**Comment by willdurand** [comment]
Not really, maybe adding a CLI would mitigate the issue but it is not urgent I'd say.

**Comment by mikbry** [comment]
okay
